pkg/commands/workload.go

```diff
@@ -390,6 +402,40 @@ func (opts *WorkloadOptions) checkToPublishLocalSource(taggedImage string, c *cl
 	return okToPush
 }
 
+func (opts *WorkloadOptions) loadExcludedPaths(c *cli.Config) []string {
+	exclude := []string{}
+	if opts.ExcludePathFile != "" {
+		p := filepath.Join(opts.LocalPath, opts.ExcludePathFile)
+		if _, err := os.Stat(p); errors.Is(err, os.ErrNotExist) {
+			return exclude
+		}
+
+		f, err := os.Open(p)
+		if err != nil {
+			c.Infof("Unable to read %s file.\n", opts.ExcludePathFile)
+			return exclude
+		}
+		defer f.Close()
+		r := bufio.NewReader(f)
+		for {
+			l, _, err := r.ReadLine()
+			if err == io.EOF {
+				break
+			}
+			p := strings.TrimSpace(string(l))
+			if len(p) == 0 || strings.HasPrefix(p, "#") {
+				continue
+			}
+			if strings.HasSuffix(p, string(os.PathSeparator)) {
```

Comment:
why is PathSeparator stripped out? 
- i think its because imgpkg is doing exact match on the name of dir, i want to confirm my understanding here

Reply:
yes, that is correct, folders suffixed with `PathSeparator` are not being excluded